package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wechaty-puppet",
-  "version": "1.19.1",
+  "version": "1.19.2",
```

Comment:
Seems like this is a breaking change, shall we update the minor version?

Reply:
I'm not sure. I think I managed to keep it compatible with previous versions.